### PR TITLE
Feature/UI

### PR DESCRIPTION
**Pull request by Terapool**

### CODE REVIEW

**Review by fooooxmr — APPROVED** [review]

File: app/views/layouts/_header.html.slim [hunk 1]
```diff
@@ -16,7 +16,7 @@ nav.navbar.navbar-expand-md.navbar-dark.bg-dark.p-1
           a.nav-link class=('active' if current_locale == menu_locales[label]) href=url_for(locale: menu_locales[label])
             = label
     - if user_signed_in?
-        = button_to "#{t('global.logout')}", destroy_user_session_path, method: :delete, :class=>"btn btn-outline-light btn-sm"
+      = button_to "#{t('global.logout')}", destroy_user_session_path, method: :delete, :class=>"btn btn-outline-light btn-sm"
```
Comment (evrusha) [inline app/views/layouts/_header.html.slim:19]: Pls use the new hash syntax

File: app/views/layouts/_header.html.slim [hunk 2]
```diff
@@ -16,7 +16,7 @@ nav.navbar.navbar-expand-md.navbar-dark.bg-dark.p-1
...
     - else
       = button_to "#{t('global.login')}", new_user_session_path, :class=>"btn btn-outline-light btn-sm"
```
Comment (evrusha) [inline app/views/layouts/_header.html.slim:21]: Same

File: app/views/layouts/_toast.html.slim [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.toast-container.position-fixed.bottom-0.end-0.p-3
+  - if !!alert
```
Comment (evrusha) [inline app/views/layouts/_toast.html.slim:2]: 'if !!alert' is the same as 'if alert'??

File: app/views/layouts/_toast.html.slim [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    div id='toast-warning'
+      .toast.bg-warning
+        .toast-header
+          div.text-dark
+            i == Octicons::Octicon.new("alert").to_svg
+            = alert
+          button.btn-close type='button' onclick="document.getElementById('toast-warning').style.display='none'"
+  - if !!notice
```
Comment (evrusha) [inline app/views/layouts/_toast.html.slim:10]: 'if !!notice' is the same as 'if notice'??

File: app/views/layouts/_toast.html.slim (hunk 3 above)
Comment (fooooxmr) [reply app/views/layouts/_toast.html.slim:2]: It's a JS habit :) In Ruby we do nor need convert value to compare it.

Comment (fooooxmr) [reply app/views/layouts/_toast.html.slim:2]: Please,  fix it :)

**Review by ritamma — APPROVED** [review]

**Review by Dmitryshl — APPROVED** [review]

**Comment by Terapool** [comment]
All comments are resolved, @evrusha, pls, approve

**Pull request merged**